pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -396,7 +396,7 @@ func (p *PriorityQueue) isPodBackingOff(pod *v1.Pod) bool {
 // backoffPod checks if pod is currently undergoing backoff. If it is not it updates the backoff
 // timeout otherwise it does nothing.
 func (p *PriorityQueue) backoffPod(pod *v1.Pod) {
-	p.podBackoff.Gc()
+	p.podBackoff.CleanupPodsNotBackingoff()
```

Comment:
Suggest to keep the name `Gc` or rename `CleanupPodsNotBackingoff()` to `CleanupPodsCompletesBackingoff()`